apps/mull-ui/src/app/app.tsx

```diff
@@ -39,24 +56,33 @@ export const App = () => {
     return {};
   };
 
+  // TODO: Replace with spinner or loading component
+  if (loading) return <div>Loading...</div>;
+
   return (
-    <UserProvider value={{ userId, setUserId, accessToken, setAccessToken }}>
+    <UserProvider value={{ userId, setUserId }}>
```

Comment:
IDK if this was already discussed or resolved, but I still don't get redirected to the login page on existing routes like /map or /asdfasdf. I think it might be better to show that 404 when you're logged in, but maybe not when you're not.

Reply:
I will address this issue.

Reply:
thanks for addressing it!